### PR TITLE
Show only verified users' applications in world data

### LINKED ISSUE #60: Do we need to check that the applications in `roles/world` belong to verified (non-anonymous) users only?
Do we need to check that the applications belong to verified (non-anonymous) users only?

_Originally posted by @zognin in https://github.com/sivayogasubramanian/caza/pull/45#discussion_r972544740_

### PR DESCRIPTION
**Pull request by ganhongyao**
Closes #60

### CODE REVIEW

**Comment by ganhongyao** [comment]
Depends on #90

**Review by sivayogasubramanian — APPROVED** [review]
LGTM! Can merge once merge conflicts are resolved.

**Pull request merged**
